### PR TITLE
Adds Scope abstract base class, adds EnergyACMeas, IrradGlobalHorizMe…

### PR DESCRIPTION
**Pull request by kevinkirkl3y**
…as, PowerACMeas, TempAmbMeas  objects.

### CODE REVIEW

File: docs/whatsnew/2202.1.0.rst [hunk 1]
```diff
@@ -29,6 +29,7 @@ Object changes
 * Added System Object (:pull:`73`)
 * Add to Installer: InstallerName (:pull:`74`)
 * Add Forecast, ForecastScope objects (:pull:`79`)
+* Add Scope, Measurement, EnergyACMeas, IrradGlobalHorizMeas, PowerACMeas, TempAmbMeas (:pull:`82`)
```
Comment (cwhanse) [inline docs/whatsnew/2202.1.0.rst:32]: We need to move this to a new file 2202.1.1.rst and not modify 2202.1.0.rst.

I'd say "Renamed ForecastScope to Scope"

Have we removed ForecastScopeType and its ItemType?

File: Master-OB-OpenAPI.json [hunk 2]
```diff
@@ -10739,6 +10739,151 @@
...
+        ]
+      },
+      "VoltageDCMeas": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/Measurement"
+          },
+          {
+            "type": "object",
+            "description": "Object containing Voltage DC measurements.",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:10835]:
```suggestion
            "description": "Object containing VoltageDC measurements.",
```

File: Master-OB-OpenAPI.json [hunk 3]
```diff
@@ -10739,6 +10739,151 @@
...
+        }
+      },
+      "EnergyACMeas": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/Measurement"
+          },
+          {
+            "type": "object",
+            "description": "An object containing Energy AC measurements.",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:10819]:
```suggestion
            "description": "An object containing EnergyAC measurements.",
```

**Review by jrippingale — APPROVED** [review]

File: docs/whatsnew/2202.1.1.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+~~~~~~~
+
+
+Documentation
+~~~~~~~~~~~~~
+
+
+Contributors
+~~~~~~~~~~~~
+* Cliff Hansen (:ghuser:`cwhanse`)
```
Comment (cwhanse) [inline docs/whatsnew/2202.1.1.rst:30]:
```suggestion
```

File: docs/whatsnew/2202.1.1.rst [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* Kevin Kirkley (:ghuser:`kevinkirkl3y`)
+* Zia Swim (:ghuser:`blizita`)
+* Taos Transue (:ghuser:`reepoi`)
```
Comment (cwhanse) [inline docs/whatsnew/2202.1.1.rst:33]:
```suggestion
```

**Review by cwhanse — APPROVED** [review]
For now, contributors are only those with PRs into the tagged version. We could change that if the group feels it should be an accumulated list.

**Pull request merged**